tests/test_simpleflow/test_canvas.py

```diff
@@ -113,6 +113,26 @@ def test_simplified_declaration(self):
         with self.assertRaises(exceptions.ExecutionBlocked):
             future.result
 
+    def test_group_with_workflow(self):
+        """ Test that it is possible to provide a WorkflowTask to a Group(). """
+
+        class ChildWorkflowGroupWithWorkflow(Workflow):
+            name = "ChildWorkflow"
+
+            def run(self, str1, *args, **kwargs):
+                return {"str1": str1, "kwargs": kwargs}
+
+        future = Group(
+            WorkflowTask(
+                None,
+                ChildWorkflowGroupWithWorkflow,
+                str1="str1",
+                **{"hello": "world"}
+            )
+        ).submit(executor)
+        self.assertTrue(future.finished)
```

Comment:
:thinking: Shouldn't one wait before checking `finished`?

Reply:
Discussed OOB: the local executor makes this correct

Reply:
As seen with you; it appears to be un-needed